src/frontend/src/eth/components/loaders/LoaderEthTransactions.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import { tokenNotInitialized } from '$eth/derived/nav.derived';
+	import { loadTransactions } from '$eth/services/transactions.services';
+	import { tokenWithFallback } from '$lib/derived/token.derived';
+	import type { TokenId } from '$lib/types/token';
+	import { isNetworkIdEthereum } from '$lib/utils/network.utils';
+
+	let tokenIdLoaded: TokenId | undefined = undefined;
+
+	const load = async () => {
```

Comment:
Just refactoring - no code changes?

Reply:
Correct, just copy/paste. But please re-review, since I included some tests, just to be sure.